test_util/README.md

```diff
@@ -2,5 +2,4 @@
 
 The dcos-integration-test harness and cloud deployment utilities that were previously hosted in this directory have been moved to: [DC/OS Test Utils](https://github.com/mesosphere/dcos-test-utils)
 
-Currently, this directory contains:
-* `run-all`: Checks out [DC/OS Vagrant](https://github.com/dcos/dcos-vagrant) and provides a DC/OS cluster from the onprem installer (`dcos_generate_config.sh`) sitting in the root of this repo (presumably generated by building DC/OS)
+Currently, this directory contains terraform helpers.
```

Comment:
If I'm not blind this directory contains nothing when `run-all` is removed.

Reply:
There are 4 other files https://github.com/dcos/dcos/tree/master/test_util
* README.md
* terraform_init.sh 	
* terraform_logs.sh
* terraform_test.sh

Reply:
Ah, I was on the wrong branch locally. All good.